### PR TITLE
Use ruby-install instead of rbenv

### PR DESCRIPTION
**Pull request by tamird**
Couple other changes while I'm here and big scary things are printed on my screen.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -25,20 +25,21 @@ RUN curl -sL https://deb.nodesource.com/setup_5.x | bash - && \
...
 	php5-cli \
-	php5-pgsql \
-	# Ruby
-	rbenv \
-	libreadline6-dev \
-	&& rm -rf /var/lib/apt/lists/*
+	php5-pgsql
 
 RUN pip install psycopg2
+RUN npm install pg
```
Comment (jordanlewis) [inline Dockerfile:31]: why'd you get rid of the version pin?

**Review by jordanlewis — COMMENTED** [review]
lgtm as long as you tested the new image against examples-orms with `make dockertest`

**Review by dt — APPROVED** [review]
did we have pg pinned in that npm install for a reason?

File: Dockerfile (hunk 1 above)
Comment (tamird) [reply Dockerfile:31]: Because I'm going to fix the issue in the main repo that required it.

**Comment by tamird** [comment]
@dt the reason is https://github.com/cockroachdb/cockroach/issues/7441

**Review by jordanlewis — APPROVED** [review]

**Pull request merged**